.mergify.yml

```diff
@@ -4,7 +4,6 @@ pull_request_rules:
       merge:
         method: rebase
         rebase_fallback: merge
```

Comment:
might be better to switch to the queue action: https://blog.mergify.com/strict-mode-deprecation/

Reply:
ack